shuup/admin/static_src/base/js/messages.js

```diff
@@ -60,13 +54,12 @@ window.Messages = (function Messages(document) {
 
         const dismissIcon = document.createElement("span");
         dismissIcon.className = "dimiss-icon";
```

Comment:
I tried to fix this typo here to `"dismiss-icon"` and change the corresponding class name in `shuup/admin/static_src/base/scss/shuup/messages.scss`, but it just wouldn't work and the X icon was not showing.